frontend/test/metabase/scenarios/models/helpers/e2e-models-helpers.js

```diff
@@ -110,3 +110,13 @@ export function startQuestionFromModel(modelName) {
   cy.findByText("Models").click();
   cy.findByText(modelName).click();
 }
+
+export const openModelActions = () => {
+  cy.findAllByTestId("question-action-buttons-container").within(() => {
+    cy.icon("ellipsis").click();
+  });
+};
+
+export const closeModelActions = () => {
+  cy.findAllByTestId("qb-header").click();
+};
```

Comment:
Should we use `findBy*` instead of `findAllBy*` here?

Reply:
I'm actually moving these to `e2e-ui-elements-helpers` so they can be used in the model, question and bookmark suites without being redefined

Reply:
But also yes. removed the `All` 😅